sound/soc/intel/boards/sdw_rt711_rt1308_rt715.c

```diff
@@ -702,14 +703,15 @@ static bool is_unique_device(const struct snd_soc_acpi_link_adr *link,
 
 static int create_codec_dai_name(struct device *dev,
 				 const struct snd_soc_acpi_link_adr *link,
-				 struct snd_soc_dai_link_component *codec)
+				 struct snd_soc_dai_link_component *codec,
+				 int offset)
 {
 	int i;
 
 	for (i = 0; i < link->num_adr; i++) {
 		unsigned int sdw_version, unique_id, mfg_id;
 		unsigned int link_id, part_id, class_id;
-		int id;
+		int id, index;
```

Comment:
can you avoid 'id' and use something meaningful?

Reply:
yes, I will fix it.